tests/__init__.py

```diff
@@ -0,0 +1,116 @@
+import tempfile
+import zipfile
+import tarfile
+import os.path
+import logging
+import pytest
+from pathlib import Path
+from pprint import pformat
+from deepdiff import DeepDiff
+
+from spark_log_parser.parsing_models.application_model_v2 import SparkApplication
+
+logging.basicConfig()
+logging.root.setLevel(logging.INFO)
```

Comment:
Interesting. If I don't do this, though, then no `stdout` logs show up for test failures, which can be very useful for determining why a test failed. Is there another way to achieve that?

Reply:
If there is a failure then shouldn't all relevant stuff be logged at the error level? Maybe I'm missing something - logging usually goes to stderr, right?

I'm all for leaving this in if it helps. Generally, you want tests to be small providing only pass/fail output. Detailed Pytest error output and the ability to quickly debug a test offsets the need for the telemetry that we rely on in production.

Reply:
yeah, the idea is that `pytest` will capture and display anything written to `stdout`/`err` in the event that a test Errors out. Some of those logs may be useful in order to tell e.g. _which_ file in an archive the test failed on. So, for example, if I force one of the test to fail and have this logging unconfigured, I get - 

<img width="954" alt="Screenshot 2022-12-20 at 2 37 59 PM" src="https://user-images.githubusercontent.com/117235708/208780156-133e52af-2bac-4b95-921a-0c2dbfb90928.png">

But if I add the logging configuration, we get instead - 
<img width="1100" alt="Screenshot 2022-12-20 at 2 39 01 PM" src="https://user-images.githubusercontent.com/117235708/208780295-459dd497-bdb2-4532-a51e-02087e9ca6e8.png">

But you are correct that following the stacktrace in the pytest output will more or less get us this information as well! Perhaps I fell back to looking at logs because I was still getting used to the pytest output